OpenRA.Mods.Common/Traits/Attack/AttackFollow.cs

```diff
@@ -81,6 +90,19 @@ public override Activity GetAttackActivity(Actor self, Target newTarget, bool al
 			return new AttackActivity(self, newTarget, allowMove, forceAttack);
 		}
 
+		public override void OnQueueAttackActivity(Actor self, Target target, bool queued, bool allowMove, bool forceAttack)
+		{
+			// If not queued we know that the attack activity will run next
+			// we can improve responsiveness for turreted actors by preempting
```

Comment:
Readability nit: It looks like this starts a new sentence (or at least should), so the 'We' should use a capital W.